schema.sql

```diff
@@ -38,3 +38,25 @@ CREATE TABLE IF NOT EXISTS public.animals
         ON UPDATE NO ACTION
         ON DELETE NO ACTION
 )
+
+--Fourth Project
+--Vets table:
+CREATE TABLE IF NOT EXISTS public.vets
+(
+    id integer NOT NULL,
+    name character varying(100) COLLATE pg_catalog."default",
+    age integer,
+    date_of_graduation date,
+    CONSTRAINT vets_pkey PRIMARY KEY (id)
+)
+--Specializations table
+CREATE TABLE IF NOT EXISTS public.species
+(
+    id integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    name character varying COLLATE pg_catalog."default",
+    CONSTRAINT species_pkey PRIMARY KEY (id)
+)
+--Visits table
+CREATE TABLE IF NOT EXISTS public.visits
+(
+)
```

Comment:
- [ ] Hello! 📋 Let's create a specialization table and a visits table with the attributes specified in the project guidelines. These changes will update your schema file, making it easier for anyone to recreate the database. 🛠️ This means that the database structure will be well-organized and straightforward for others to use. 🤝 So, let's get started with these two essential tables to ensure seamless database management!